### ISSUE TITLE
touch: cannot touch ‘./config_files.txt’: Permission denied

### ISSUE THREAD

**Issue by trevor** [phusion/passenger#1541]
passenger 5.0.10
nginx 1.8
ruby 2.1.2p95

This has been appearing in the passenger log:
```
touch: cannot touch ‘./config_files.txt’: Permission denied
touch: cannot touch ‘./web_server.txt’: Permission denied
```

Content of tmp directory:
```
total 40
drwxr-xr-x 2 www-data www-data 4096  agents.s
drwx--x--x 2 www-data www-data 4096  apps.s
-rw-r--r-- 1 root     root       26  config_files.txt
-rw-r--r-- 1 www-data root        5  control_process.pid
-rw-r--r-- 1 www-data www-data    0  creation_finalized
-rw------- 1 www-data www-data   24  full_admin_password.txt
-rw-r--r-- 1 www-data www-data    0  lock
-rw-r--r-- 1 www-data www-data  266  properties.json
-rw------- 1 www-data www-data   24  read_only_admin_password.txt
-rw-r--r-- 1 www-data www-data    5  server.pid
-rw-r--r-- 1 www-data www-data    5  watchdog.pid
-rw-r--r-- 1 root     root       11  web_server.txt
```

Note those files appear to be owned by root.

Relevant lines from nginx config:
```
user www-data;

http {
    passenger_root /var/lib/gems/2.1.0/gems/passenger-5.0.10;
    passenger_ruby /usr/bin/ruby;
    passenger_default_user www-data;
    passenger_default_group www-data;
    passenger_user_switching off;
    [...]
}
```

**Comment by FooBarWidget** [comment]
The messages are annoying, but non-fatal.

We'll have a look at it.

**Comment by FooBarWidget** [comment]
Fixed in 5.0.11.
